config/default/common/config/metadata/stories/fires_overview_2019/step002.md

```diff
@@ -0,0 +1,5 @@
+<p>Data collected by the satellites utilize an algorithm that exploits the strong emission of mid-infrared radiation from fires and thermal anomalies - these fires and thermal anomalies are shown here as red points.  The points represent the center of a pixel within which one or multiple fires have occurred. For the MODIS instrument, the point represents the center of a 1km pixel; for the VIIRS instrument, the point represents the center of a 375m pixel. Using the eye icon, turn on and off the MODIS and VIIRS fire points to view the differences between the sources.
+</p>
+Find out more in the FIRMS FAQ:<br>
+[What does a MODIS active fire detection mean on the ground?](https://earthdata.nasa.gov/faq/firms-faq#ed-modis-fire-onground);<br>
```

Comment:
Remove the semi-colon at the end of this line.

Reply:
done